<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #FFCC21;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="cbxUsers">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>400</y>
      <width>314</width>
      <height>56</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #2C618B;
color: rgb(255, 255, 255);
padding-left: 10px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblIngresoAlSistema">
    <property name="geometry">
     <rect>
      <x>502</x>
      <y>0</y>
      <width>361</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Ingreso al Sistema</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="imgSalidaDelSistema">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>10</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #2C618B;
color: rgb(255, 255, 255);
border:none;
border-radius: 10px;
border-left: 1px solid #01304A;
border-right: 1px solid #01304A;
border-bottom: 5px solid #01304A;
padding: 5px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>480</y>
      <width>56</width>
      <height>56</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #01304A;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="imgUser">
    <property name="geometry">
     <rect>
      <x>513</x>
      <y>400</y>
      <width>56</width>
      <height>56</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
background-color: #01304A;
padding: 15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSalirDelSistema">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>10</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbltexto">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>550</y>
      <width>461</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color: #242424;</string>
    </property>
    <property name="text">
     <string>SELECCIONA TU USUARIO Y ESCRIBE TU CONTRASEÑA</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="imgLogoBalinsa">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>660</y>
      <width>320</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="imgIngresoAlSistema">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>0</y>
      <width>450</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="imgCirculoIngresoAlSistema">
    <property name="geometry">
     <rect>
      <x>593</x>
      <y>110</y>
      <width>180</width>
      <height>180</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 90px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="imgPassword">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>480</y>
      <width>56</width>
      <height>56</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
background-color: #01304A;
padding: 15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>400</y>
      <width>56</width>
      <height>56</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #01304A;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="contenedorInicioSesion">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>210</y>
      <width>535</width>
      <height>395</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 40px;
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAccederSistema">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>580</y>
      <width>220</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #01304A;
    color: rgb(255, 255, 255);
    border: none;
    border-radius: 15px;
    border-left: 1px solid #011F2F;
    border-right: 1px solid #011F2F;
    border-bottom: 5px solid #011F2F;
}

QPushButton:hover {
    background-color: #023E5E;
}

QPushButton:pressed {
    background-color: #062B3F;
}</string>
    </property>
    <property name="text">
     <string>ACCEDER</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblSistemaParaPesajeVehicular">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>300</y>
      <width>535</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat ExtraBold</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color: #242424;</string>
    </property>
    <property name="text">
     <string>SISTEMA PARA 
 PESAJE VEHICULAR</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtUsers">
    <property name="geometry">
     <rect>
      <x>515</x>
      <y>480</y>
      <width>314</width>
      <height>56</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #2C618B;
color: rgb(255, 255, 255);
border-top-left-radius: 10px;
border-bottom-left-radius: 10px;
padding-left: 10px;</string>
    </property>
    <property name="placeholderText">
     <string>Contraseña</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>730</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>© 2024 Desarrollado por Jhon Huertas</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>730</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>© 2024 Diseñado por Leonardo Garcia Jimenez</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <zorder>contenedorInicioSesion</zorder>
   <zorder>imgSalidaDelSistema</zorder>
   <zorder>label_2</zorder>
   <zorder>btnSalirDelSistema</zorder>
   <zorder>lbltexto</zorder>
   <zorder>imgLogoBalinsa</zorder>
   <zorder>imgIngresoAlSistema</zorder>
   <zorder>label</zorder>
   <zorder>btnAccederSistema</zorder>
   <zorder>lblSistemaParaPesajeVehicular</zorder>
   <zorder>txtUsers</zorder>
   <zorder>imgCirculoIngresoAlSistema</zorder>
   <zorder>cbxUsers</zorder>
   <zorder>imgUser</zorder>
   <zorder>imgPassword</zorder>
   <zorder>lblIngresoAlSistema</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
